bcdaworker/main_test.go

```diff
@@ -137,11 +137,28 @@ func TestAppendErrorToFile(t *testing.T) {
 	os.Remove(filePath)
 }
 
-func (bbc *MockBlueButtonClient) GetExplanationOfBenefitData(patientID string) (string, error) {
-	return eobData, nil
+// Returns copy of a static json file (From Blue Button Sandbox originally) after replacing the patient ID of 20000000000001 with the requested identifier
+
+func (bbc *MockBlueButtonClient) GetData(path string, params url.Values) (string, error) {
+	var identifier string
+	if strings.Contains(path, "Coverage") {
+		identifier = "beneficiary"
+	} else if strings.Contains(path, "Patient") {
+		identifier = "_id"
+	} else if strings.Contains(path, "ExplanationOfBenefits") {
```

Comment:
Should this be singular ("ExplanationOfBenefit")?

Reply:
yes.  FIxing.